sdv/sampler.py

```diff
@@ -405,6 +407,75 @@ def unflatten_model(self, parent_row, table_name, parent_name):
 
         return self.modeler.model.from_dict(model_parameters)
 
+    def __get_missing_valid_rows(self, synthesized, drop_indices, valid_rows, num_rows):
```

Comment:
Why is this method prefixed with a double underscore?

Reply:
This was a typo